tools/profiler.md

```diff
@@ -24,24 +22,18 @@ Regarding latency, it is a whole topic in itself, but remember that we are talki
 
 ## Prerequisites
 
-Starting from Unity 2021.1, the Unity profiler can extend its features with Netcode for GameObjects [NetCode]. Netcode enables you to profile your game in the same tool you are already using for CPU, GPU, and memory.
-
-To enable profiling support for Netcode, install the `com.unity.multiplayer.tools` package from the [Unity Package Manager](../tools/install-tools.md). Then, you should see two additional modules in your profiler (**Window** > **Analysis** > **Profiler**):
-* Messages - Network messages sent to the transport
-* Objects - Activity related to, and grouped by, network objects
+Starting from Unity 2021.1, the Unity profiler can extend its features with Netcode for GameObjects [Netcode]. Netcode enables you to profile your game in the same tool you are already using for CPU, GPU, and memory.
 
-:::important
-If you have the profiler window open when you install the tools package in 2020.3 or 2021.1, you may need to close and reopen the profiler window for the moduels to appear.
+:::note
+The network profiler samples its data by frame rate. Therefore, you may find the graphs a little spiky.
```

Comment:
Maybe this note should be moved after the installation instructions? I think those might be more important to front-load.

Reply:
👍